packages/twenty-front/src/modules/prefetch/hooks/useIsFavoriteFoldersPrefetchLoading.ts

```diff
@@ -0,0 +1,11 @@
+import { prefetchIsLoadedFamilyState } from '@/prefetch/states/prefetchIsLoadedFamilyState';
+import { PrefetchKey } from '@/prefetch/types/PrefetchKey';
+import { useRecoilValue } from 'recoil';
+
+export const useIsFavoriteFoldersPrefetchLoading = () => {
```

Comment:
this hook is useless we can call the state directly (states are part of the public api of a module, no need to create a hook just to wrap it)